templates/master/00-master/sno/files/sno-forcedns-fix.yaml

```diff
@@ -0,0 +1,12 @@
+mode: 0755
```

Comment:
sno is not a platform. So, I am curious how this file is getting only added to sno cluster?

Reply:
It was added as a platform previously, i mean we have a check that add files if it is sno.
https://github.com/openshift/machine-config-operator/blob/768fe5f4952094525ee8981254a933dd846e6a2f/pkg/controller/template/render.go#L244

Reply:
ah interesting. Missed that change.